using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -103,116 +119,150 @@ This API synchronizes the provided customer identifier type key and value with t
 ```
 
-#### **Examples**
+o   *identifiers* is a dictionary that contains IDs with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
+
+o   *authenticationState* (VisitorIDAuthenticationState) value indicating authentication state for the user contaisn one of the VisitorID.AuthenticationState values `ACPMobileVisitorAuthenticationStateAuthenticated`,`ACPMobileVisitorAuthenticationStateLoggedOut` and `ACPMobileVisitorAuthenticationStateUnknown`
+
+#### Examples
 
 **Objective-C**
 
 ```objectivec
-NSDictionary *ids = @{@"idType":@"idValue"};
-[ACPIdentity syncIdentifiers:ids];
+NSDictionary *ids = @{@"idType1":@"idValue1", 
+                      @"idType2":@"idValue2", 
+                      @"idType3":@"idValue3"};
+[ACPIdentity syncIdentifiers:ids authentication:ACPMobileVisitorAuthenticationStateAuthenticated];
 ```
 
 **Swift**
 
 ```swift
-let identifiers : [String: String] = ["idType1":"idValue1", "idType2":"idValue2"];
-ACPIdentity.syncIdentifiers(identifiers)
+let identifiers : [String: String] = ["idType1":"idValue1",
+                                      "idType2":"idValue2",
+                                      "idType3":"idValue3"];
+ACPIdentity.syncIdentifiers(identifiers, authentication:
+ACPMobileVisitorAuthenticationState.authenticated)
 ```
 
-## syncIdentifiers \(overloaded\) <a id="syncidentifiers-overloaded"></a>
-
-Updates the provided customer IDs with the Adobe Experience Cloud ID Service.
+### syncIdentifiers \(overloaded\) 
 
-This API synchronizes the provided customer identifiers to the Adobe Experience Cloud ID Service. If a customer ID type matches an existing ID type, the customer ID is updated with the new ID value and [authentication state](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#authenticationstate). New customer IDs are added. These IDs are preserved between app upgrades, are saved and restored during the standard application backup process, and are removed at uninstall. If the current SDK privacy status is `optedout`, calling this method results in no operations being performed.
+This  API  `syncIdentifiers()` is an overloaded version that does not include the parameter for the Authentication State, and assumes a default value as `ACPMobileVisitorAuthenticationStateUnknown`
 
-{% hint style="info" %}
-The `identifiers` dictionary contains IDs with the Identifier type as the key, and the string identifier as the value.
-{% endhint %}
-
-### **Syntax**
+#### Syntax
 
 ```objectivec
-+ (void) syncIdentifiers: (nullable NSDictionary*) identifiers          
-         authentication: (ACPMobileVisitorAuthenticationState) authenticationState;
++ (void) syncIdentifiers: (nullable NSDictionary*) identifiers;
 ```
 
-### **Examples**
```

Comment:
- Please move the parameters after the "Example" section.
- Please remove the "o" and keep the asterisk for the bullet point.
- Move the 2nd sentence to a new line under the bullet point and reword as follows:
"If the identifier type and identifier do not have non-null or empty values, the identifier is ignored by the Identity extension."